x/mint/keeper/keeper_test.go

```diff
@@ -59,6 +63,16 @@ func TestKeeperTestSuite(t *testing.T) {
 	suite.Run(t, new(KeeperTestSuite))
 }
 
+func (suite *KeeperTestSuite) TestGRPCParams() {
+	_, _, queryClient := suite.App, suite.Ctx, suite.queryClient
+
+	_, err := queryClient.Params(gocontext.Background(), &types.QueryParamsRequest{})
+	suite.Require().NoError(err)
+
+	_, err = queryClient.EpochProvisions(gocontext.Background(), &types.QueryEpochProvisionsRequest{})
+	suite.Require().NoError(err)
+}
+
 func (suite *KeeperTestSuite) TestMintCoinsToFeeCollectorAndGetProportions() {
 	mintKeeper := suite.App.MintKeeper
```

Comment:
Also, let's update the `hooks_test.go` to utilize the `KeeperTestSuite` please

Reply:
should be good now! Also why are `hooks.go` and `hooks_test.go` files under keeper in mint module? For every other module it's under types?

Reply:
I might be wrong but I think there are 2 types of hooks:
- The kind that module implements
    * defined in keeper
    * Example: incentives implements epoch hooks: https://github.com/osmosis-labs/osmosis/blob/3be443a02a5fc858a63bf8acd29654a58583dd32/x/incentives/keeper/hooks.go#L14
- The kind that module defines
    * defined in types
    * incentives defines hooks: https://github.com/osmosis-labs/osmosis/blob/3be443a02a5fc858a63bf8acd29654a58583dd32/x/incentives/types/hooks.go#L5

I think the location for mint hooks is correct because mint keeper implemented epoch end hooks